lib/src/signed_video_h26x_auth.c

```diff
@@ -114,6 +114,24 @@ decode_sei_data(signed_video_t *self, const uint8_t *payload, size_t payload_siz
   return status;
 }
 
+/**
+ * Detects if there are any missing SEI messages based on the GOP counter and updates the GOP state.
+ */
+static void
+detect_sei_loss_and_sync_gop(signed_video_t *self)
```

Comment:
I find this function name misleading. The name indicates that the GOP is sync'ed, but that is not true. The only thing done is to detect a lost SEI. How about renaming to `detect_lost_sei`